flamegraph.pl

```diff
@@ -682,16 +685,21 @@ sub flow {
 		}
 		
 		t.textContent = txt;
+
 		// Fit in full text width
-		if (/^ *\$/.test(txt) || t.getSubStringLength(0, txt.length) < w)
+		if (/^ *\$/.test(txt) || t.getComputedTextLength() < w)
```

Comment:
@cwarden indeed, in my tests `t.getComputedTextLength()` is the most expensive function. OTOH the text lengths in this case are not too big. Can you PR me your version for testing?